File: 462-minimum-moves-to-equal-array-elements-ii/462-minimum-moves-to-equal-array-elements-ii.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var minMoves2 = function(nums){
    nums.sort((a,b)=>a-b);
    let n=nums.length;
    let mid=Math.floor(n/2);
    let step=0;
    for(let i=0;i<n;i++){
        step+=Math.abs(nums[mid]-nums[i]);
    }
    return step;
};

File: 462-minimum-moves-to-equal-array-elements-ii/462-minimum-moves-to-equal-array-elements-ii.cpp
class Solution {
public:
    int minMoves2(vector<int>& nums) {
        sort(nums.begin(),nums.end());
        int mid=nums[nums.size()/2];
        int count=0;
        for(auto &it:nums)
        {
            count+=abs(it-mid);
        }
        return count;
   }
};